src/handbook/marketing/messaging.md

```diff
@@ -85,57 +85,23 @@ FlowFuse is the open-source Industrial Application Platform.
 
 ### FlowFuse in one sentence - Under 150 characters
 
-FlowFuse empowers teams to integrate digital systems with their physical environments, automating workflows through a low-code, end-to-end platform.
+FlowFuse is the fastest, most secure way to build industrial applications and operate at scale.
```

Comment:
I don't think this is an improvement. Technology is always making things faster, and security is not something we sell mainly. 

The buyer persona also doesn't think of themselves as applications developers?